extension/war/src/main/webapp/groovy/social/webui/profile/UIUserActivitiesDisplay.gtmpl

```diff
@@ -47,7 +47,6 @@
 
   //
   def refreshTooltipButton = _ctx.appRes("UIActivity.label.Refresh");
-  def composer_Hint_Text = _ctx.appRes("UIActivity.label.Composer_Hint_Text");
```

Comment:
Shouldn't we delete the label "UIActivity.label.Composer_Hint_Text" in the translation files ?